src/templates/index.pug

```diff
@@ -8,6 +14,9 @@ body.layout
     // Page header
     nav.layout__header#header
         a.layout__header__menu(class=['logo'])="C++ Tour"
+        div.layout__header__icons
```

Comment:
Ok sorry we seems to have misunderstanding.
The thing I wanted is `.layout__header__icons` to control behavior of `bg__image`.
But now I think I'll leave it up to you, do what you think is right :)

Also please keep in mind BEM style naming http://getbem.com/naming/

Reply:
Understood. I'll submit the changes tomorrow. :)